Lara-JS/package.json

```diff
@@ -21,7 +21,7 @@
     "tsconfig.json"
   ],
   "scripts": {
-    "build": "node scripts/copy-folder.js -i src-api/libs -o api/libs && npx tsc -b src-api src-code",
+    "build": "node scripts/copy-folder.js -i src-api/libs -o api/libs && node scripts/copy-folder.js -i src-api/visualization/public -o api/visualization/public && npx tsc -b src-api src-code",
```

Comment:
Is there a reason for the visualization API being in its own namespace (i.e., `src-api/visualization`) instead of being a LARA API (i.e. `src-api/lara/visualization`)?